### PR TITLE
Update painless-statements with if/else example

### PR DESCRIPTION
**Pull request by renshuki**
I struggled to find a conditional statement (if/else) in the Painless documentation. I think it would be nice to at least have one here. 

There is also the possibility to add somethig similar to [A Brief Painless Walkthrough](https://www.elastic.co/guide/en/elasticsearch/painless/current/painless-walkthrough.html#painless-walkthrough) but this is less generic.

Thanks.

### CODE REVIEW

**Review by jrodewig — APPROVED** [review]
LGTM, but I'll wait until a dev has a chance to review before merging.

Note: If we add this example, it probably makes sense to add other examples (such as `while`).

**Review by jdconrad — APPROVED** [review]
LGTM. Thanks for adding this.

**Comment by renshuki** [comment]
Awesome, thanks for the quick merge.
Noted I'll work on adding other statements to the doc as well @jrodewig

**Pull request merged**
